.gitignore

```diff
@@ -24,3 +24,4 @@ coverage
 build/
 dist/
 
+public/bundle.js
```

Comment:
Disregard if I am wrong but I wonder if your .gitignore should have also been ignored upon push.

Reply:
Fixed!